data/migrations/20190323112606_createProjectsActionsTable.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('projects_actions', tbl => {
+    tbl.increments()
+    tbl.integer('project_id').notNullable()
+    tbl.integer('action_id').notNullable()
```

Comment:
This table is questionable since an action only ever belongs to one project per the spec.